src/Servers/HttpSys/test/FunctionalTests/Http2Tests.cs

```diff
@@ -470,6 +470,80 @@ public async Task ConnectionClose_AdditionalRequests_ReceivesSecondGoAway()
                 .Build().RunAsync();
         }
 
+        [ConditionalFact]
+        [MinimumOSVersion(OperatingSystems.Windows, WindowsVersions.Win10_19H2, SkipReason = "GoAway support was added in Win10_19H2.")]
+        public async Task ConnectionRequestClose_AdditionalRequests_ReceivesSecondGoAway()
```

Comment:
You've copied more complex test. You can cover this new behavior by copying the simpler test ConnectionClose_OSSupport_SendsGoAway.